crates/wit-component/src/encoding.rs

```diff
@@ -1398,9 +1430,27 @@ impl ComponentEncoder {
         // Merge the adapter's document into our own document to have one large
         // document, but the adapter's world isn't merged in to our world so
         // retain it separately.
```

Comment:
Is this comment still correct given these changes?

Reply:
Oops, thanks!